manage.py

```diff
@@ -461,6 +461,76 @@ def delete_guid_based_uploads():
             app.file_service.system.delete(key_using_guid)
 
 
+@manager.command
+def delete_all_measures_except_two_per_subtopic():
+    """Delete a large proportion of records from our database by dropping all except the first two measures in each
+    subtopic, and all of the records associated with those dropped measures. This is used to keep our review app
+    databases under 10k rows, which is (as of 2019-04-01) the limit on Heroku."""
+    from application.utils import TimedExecution
+    from application.cms.models import (
+        Subtopic,
+        Measure,
+        DataSource,
+        DataSourceInMeasureVersion,
+        MeasureVersion,
+        Dimension,
+        Chart,
+        Table,
+    )
+
+    with TimedExecution("Delete 3rd measure and onward in each subtopic"):
```

Comment:
Maybe we could put the failsafe in like we do for `pull_prod_data`,because this is terrifying. 
ie:
```
    environment = os.environ.get("ENVIRONMENT", "PRODUCTION")
    if environment.upper() == "PRODUCTION":
        print("It looks like you are running this in production.")
        print("Do not run this command in this environment as it deletes data")
        sys.exit(-1)
```

Reply:
Yep, I definitely meant to add this and forgot. Well spotted/rememered.